containers/filters/AddFilterModal.js

```diff
@@ -38,12 +38,15 @@ function AddFilterModal({ filter, type, mode = 'create', onEdit = noop, ...props
     const reqUpdate = useApiWithoutResult(updateFilter);
 
     const create = async (filter) => {
-        const { Filter } = await reqCreate.request(filter);
-        call();
-        createNotification({
-            text: c('Notification').t`${Filter.Name} created`
-        });
-        props.onClose();
+        try {
+            const { Filter } = await reqCreate.request(filter);
+            createNotification({
+                text: c('Notification').t`${Filter.Name} created`
+            });
+        } finally {
+            call();
```

Comment:
If the request fail we should not call the event manager since there is no filter model to synchronize.

Reply:
It's what I thought at the beginning but in fact yes.

In fact, the filter is added but not activated and you're not allow to activate it. But it's added so without that, it's not shown.

Reply:
It sounds like a bug, you should check the issue with @bartbutler.

Reply:
I just checked with @bartbutler.

It works as expected. The limit is on the number of active filters not "existing" filters. This way, someone spending time to configure a new filter doesn't loose his work.

Reply:
@Swiip could you add a comment in the code explaining why we are calling `call` from the event manager even if the request fail please?